components/feature/addons/src/main/res/values-kn/strings.xml

```diff
@@ -104,22 +130,56 @@
+    <string name="mozac_feature_addons_not_yet_supported_caption2">ಶಿಫಾರಸು ಮಾಡಲಾದ ವಿಸ್ತರಣೆಗಳ ಆರಂಭಿಕ ಆಯ್ಕೆಗಾಗಿ ನಾವು ಪ್ರಸ್ತುತ ಬೆಂಬಲವನ್ನು ನಿರ್ಮಿಸುತ್ತಿದ್ದೇವೆ.</string>
+    <!-- This is the caption for the add-on installation progress overlay -->
+    <string name="mozac_add_on_install_progress_caption">ಆಡ್-ಆನ್ ಅನ್ನು ಡೌನ್‌ಲೋಡ್ ಮಾಡಲಾಗುತ್ತಿದೆ ಮತ್ತು ಪರಿಶೀಲಿಸಲಾಗುತ್ತಿದೆ…</string>
+    <!-- Error shown when something unexpected happened while trying to get the add-on list from the server -->
+    <string name="mozac_feature_addons_failed_to_query_add_ons">ಆಡ್-ಆನ್‌ಗಳನ್ನು ಹುಡುಕಲುಲು ವಿಫಲವಾಗಿದೆ!</string>
     <!-- Text shown after successfully installed an add-on. %1$s is the add-on name. -->
     <string name="mozac_feature_addons_successfully_installed">%1$s ಅನ್ನು ಯಶಸ್ವಿಯಾಗಿ ಸ್ಥಾಪಿಸಲಾಗಿದೆ</string>
+    <!-- Text shown after failed to install an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_failed_to_install">%1$s ಅನ್ನು ಸ್ಥಾಪಿಸಲು ವಿಫಲವಾಗಿದೆ</string>
+    <!-- Text shown after successfully enabled an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_successfully_enabled">%1$s ಅನ್ನು ಯಶಸ್ವಿಯಾಗಿ ಸಕ್ರಿಯಗೊಳಿಸಲಾಗಿದೆ</string>
+    <!-- Text shown after failed to enable an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_failed_to_enable">%1$s ಅನ್ನು ಸಕ್ರಿಯಗೊಳಿಸಲು ವಿಫಲವಾಗಿದೆ</string>
+    <!-- Text shown after successfully disabled an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_successfully_disabled">%1$s ಅನ್ನು ಯಶಸ್ವಿಯಾಗಿ ನಿಷ್ಕ್ರಿಯಗೊಳಿಸಲಾಗಿದೆ</string>
+    <!-- Text shown after failed to disable an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_failed_to_disable">%1$s ಅನ್ನು ನಿಷ್ಕ್ರಿಯಗೊಳಿಸಲು ವಿಫಲವಾಗಿದೆ</string>
+    <!-- Text shown after successfully uninstalled an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_successfully_uninstalled">%1$s ಅನ್ನು ಯಶಸ್ವಿಯಾಗಿ ಸ್ಥಾಪಿಸಲಾಗಿದೆ</string>
+    <!-- Text shown after failed to uninstall an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_failed_to_uninstall">%1$s ಅನ್ನು ಸ್ಥಾಪಿಸಲು ವಿಫಲವಾಗಿದೆ</string>
+    <!-- Text shown after successfully removed an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_successfully_removed" tools:ignore="UnusedResources">%1$s ಅನ್ನು ಯಶಸ್ವಿಯಾಗಿ ತೆಗೆದುಹಾಕಲಾಗಿದೆ</string>
+    <!-- Text shown after failed to remove an add-on. %1$s is the add-on name. -->
+    <string name="mozac_feature_addons_failed_to_remove" tools:ignore="UnusedResources">%1$s ತೆಗೆದುಹಾಕಲು ವಿಫಲವಾಗಿದೆ</string>
+    <!-- Label shown to indicate that the add-on was migrated from a previous version of the app. %1$s is the app name most of the case it will be Firefox. -->
+    <string name="mozac_feature_addons_migrated_from_a_previous_version_label" tools:ignore="UnusedResources">ಈ ಆಡ್-ಆನ್ ಅನ್ನು ಹಿಂದಿನ% %1$s ಆವೃತ್ತಿಯಿಂದ ಸ್ಥಳಾಂತರಿಸಲಾಗಿದೆ</string>
```

Comment:
@Delphine Task cluster is not happy here neither.

```
[task 2020-07-17T00:19:25.493Z]   /builds/worker/checkouts/src/components/feature/addons/src/main/res/values-kn/strings.xml:162: Error: Incorrect formatting string mozac_feature_addons_migrated_from_a_previous_version_label; missing conversion character in '% %' ? [StringFormatInvalid]
[task 2020-07-17T00:19:25.493Z]       <string name="mozac_feature_addons_migrated_from_a_previous_version_label" tools:ignore="UnusedResources">ಈ ಆಡ್-ಆನ್ ಅನ್ನು ಹಿಂದಿನ% %1$s ಆವೃತ್ತಿಯಿಂದ ಸ್ಥಳಾಂತರಿಸಲಾಗಿದೆ</string>
[task 2020-07-17T00:19:25.493Z]       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[task 2020-07-17T00:19:25.493Z]   /builds/worker/checkouts/src/components/feature/addons/src/main/res/values-kn/strings.xml:162: Error: Inconsistent number of arguments in formatting string mozac_feature_addons_migrated_from_a_previous_version_label; found both 1 and 0 [StringFormatCount]
[task 2020-07-17T00:19:25.493Z]       <string name="mozac_feature_addons_migrated_from_a_previous_version_label" tools:ignore="UnusedResources">ಈ ಆಡ್-ಆನ್ ಅನ್ನು ಹಿಂದಿನ% %1$s ಆವೃತ್ತಿಯಿಂದ ಸ್ಥಳಾಂತರಿಸಲಾಗಿದೆ</string>
[task 2020-07-17T00:19:25.493Z]       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[task 2020-07-17T00:19:25.493Z]       /builds/worker/checkouts/src/components/feature/addons/src/main/res/values/strings.xml:167: Conflicting number of arguments here
```

Reply:
I've fixed the error in the "mozac_feature_addons_migrated_from_a_previous_version_label".
Is there another error that I can't see from your screenshot above? (for strings.xml:167?)
Couldn't find it if there is